.github/renovate.json5

```diff
@@ -1,23 +1,10 @@
 {
   "$schema": "https://docs.renovatebot.com/renovate-schema.json",
   "extends": [
-    "config:recommended"
+    "config:recommended",
+    "github>gardener/ci-infra//config/renovate/automerge-with-tide.json5"
```

Comment:
```suggestion
    "github>gardener/ci-infra/config/renovate/automerge-with-tide.json5"
```

Reply:
I believe this is correct as shown in the [docs](https://docs.renovatebot.com/config-presets/#github) under `GitHub with preset name and path`

Reply:
crazy 😮